php-files/edit/EditProgram.php

```diff
@@ -10,7 +10,7 @@
 
 <div class="row">
     <div class="form-group">
-        <div class="col-lg-4">
+        <div class="col-lg-10">
```

Comment:
i suggest we switch this out to a percentage that way it will be responsive.  and if you reference that variable on every one of these we'll only have to mess around with the width setting in one spot

Reply:
Sounds good to me. These pages just weren't loading in the search bar right. Giving them that extra length made them look nicer for the photo sharing. Each value was different, 3s and 4s I changed. Some others were longer.

Reply:
ahhhhh gotcha gotcha alright man sounds good